pkg/metrics/grafana/generate_json.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 go install github.com/google/go-jsonnet/cmd/jsonnet@latest
-git clone https://github.com/grafana/grafonnet-lib.git
+git clone https://github.com/nolouch/grafonnet-lib.git
```

Comment:
Can you explain what does this fork fix? Why only resource control page is impacted？ Will we depend on this fork forever?

Reply:
update it in comments